util/lambda_trigger/lambda_trigger.go

```diff
@@ -70,7 +70,7 @@ func triggerGithubWorkflow(event *ReleaseEvent) error {
 	// Create dispatch event https://docs.github.com/en/rest/reference/repos#create-a-repository-dispatch-event
 	workflowEndpoint := "https://api.github.com/repos/hashicorp/homebrew-tap/dispatches"
 	cask := isCask(event.Product)
-	postBody := fmt.Sprintf("{\"event_type\": \"version-updated\", \"client_payload\":{\"name\":\"%s\",\"version\":\"%s\",\"cask\":\"%s\"}}", event.Product, event.Version, cask)
+	postBody := fmt.Sprintf("{\"event_type\": \"version-updated\", \"client_payload\":{\"name\":\"%s\",\"version\":\"%s\",\"cask\":\"%t\"}}", event.Product, event.Version, cask)
```

Comment:
FWIW it's common to use structs with `json` tags in them, as opposed to `fmt` functions when rendering JSON in Go. It also makes the code (although more verbose) easier to read as you don't need to worry about escaping and formatting.

e.g.

```go
type Body struct {
	EventType     string  `json:"event_type"`
	ClientPayload *Payload `json:"client_payload"`
}

type Payload struct {
	Name    string `json:"name"`
	Version string `json:"version"`
	IsCask  bool   `json:"cask"`
}

data := &Body{
	EventType: "version-updated",
	ClientPayload: &Payload{
		Name:    "terraform",
		Version: "1.0.0",
		IsCask:  true,
	},
}
postBodyBytes, err := json.Marshal(data)
if err != nil {
	return err
}
```

Reply:
Yeah that's a good point @radeksimko , I'll do this next time I'm touching this code